flink-test-utils-parent/flink-test-utils-junit/pom.xml

```diff
@@ -21,6 +21,10 @@ under the License.
 	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
 
 	<modelVersion>4.0.0</modelVersion>
+	<properties>
+		<maven.compiler.source>8</maven.compiler.source>
+		<maven.compiler.target>8</maven.compiler.target>
+	</properties>
```

Comment:
Why is this needed? I am not sure whether we want to do this because usually all modules should work with java 8 and java 11

Reply:
This should be deleted (I added this line for local debugging purposes), sorry for importing this one.